public/icons.js

```diff
@@ -25,8 +25,13 @@ export default {
   home: {
     path: (
       <g>
-        <path id='Path_34' data-name='Path 34' d='M14.544,29.657V20.249h6.272v9.408h7.84V17.112h4.7L17.681,3,2,17.112H6.7V29.657Z' transform='translate(1.136 1.704)' fill='#4d4d4d' />
-        <path id='Path_35' data-name='Path 35' d='M0,0H37.633V37.633H0Z' fill='none' />
+        <path
+          data-name='Path 34'
+          d='M14.544,29.657V20.249h6.272v9.408h7.84V17.112h4.7L17.681,3,2,17.112H6.7V29.657Z'
+          transform='translate(1.136 1.704)'
+          fill='currentColor'
```

Comment:
What is current color?

Reply:
currentColor is a CSS value that will use whatever the current font color is, so the icon is always the same color as the text:
https://css-tricks.com/currentcolor/